### ISSUE TITLE
low performance

### ISSUE THREAD

**Issue by AlanSwift** [facebookresearch/grounded-video-description#14]
I trained the grounded video caption model  in my own experimental environment with the hyper-parameters, datasets, features, proposals and codes given. But I get a low CIDEr performance.
In the paper, we are expected to get CIDEr more than 45, but in fact, I only get 18 after 15 epochs. I think the model is converged because the CIDEr is stable in the validation dataset.
I think the difference between the test results and the performance of the paper is too large. This really bothers me. Are there any problems doing this experiment?
Thanks for your time.

**Comment by LuoweiZhou** [comment]
@AlanSwift Could you provide more details regarding your experiments? Such as the commands you were running (this includes the number of GPUs, learning rate, batch size etc.).

**Comment by AlanSwift** [comment]
I trained the model using 4 1080Ti GPU, batch_size = 120, learning_rate = 0.001(adam), att2_loss = 0.5, ground_loss = 0.1, cls_loss = 0.5 with attn_model = topdown
And I have encountered bottlenecks on IO(the dataloader suspended), so I reduce the feature dim by PCA, and related pretrained matrixs are transformed by PCA's transform matrix. I think this may not be the key reason to such low performance.

**Comment by LuoweiZhou** [comment]
@AlanSwift here are a few suggestions:
i) Since your batch size is 1/2 of the original one, a better learning_rate to start with is  5e-4/2 (so between 2e-4 or 3e-4). The one you're using might lead to a suboptimal convergence.
ii) To ensure the feature PCA reduction does not affect the pre-trained matrices, you could, for example, try visualizing the transformed [`vis_classifiers`](https://github.com/facebookresearch/grounded-video-description/blob/master/misc/model.py#L209) to see if concepts with similar semantics are still closer to each other. Also, you might want to tweak on the [fine-tuning](https://github.com/facebookresearch/grounded-video-description/blob/master/main.py#L656) learning rate. We could dive deeper into this part if you provide more details on the changes you made related to PCA.

**Comment by AlanSwift** [comment]
@LuoweiZhou Thank you very march. I will try it these days.
